### PR TITLE
Sentiment scorer for stack analysis

### PR DESCRIPTION
**Pull request by harjinder-hari**

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -16,3 +16,6 @@ pyjwt
 requests-futures
 # we require f8a_worker, but it is installed manually as it is not available in PyPI
 #f8a_worker
+
+google-cloud
+joblib
```
Comment (fridex) [inline requirements.in]: The same as in https://github.com/fabric8-analytics/fabric8-analytics-jobs/pull/100#discussion_r127678657. It should probably go to worker unless these deps are not directly used by server.

File: Dockerfile [hunk 2]
```diff
@@ -49,3 +49,5 @@ RUN sh /tmp/update_selinon.sh
 RUN cd /tmp/install_deps/ && /tmp/install_deps/apply_patches.sh
 
 RUN yum install -y npm && npm install -g semver-ranger
+
+RUN pip3 uninstall -y protobuf && pip3 install packaging appdirs && pip3 install --upgrade --no-binary :all: protobuf==3.3.0
```
Comment (msrb) [inline Dockerfile]: I am puzzled by this line, why is this needed here?

File: requirements.in (hunk 1 above)
Comment (msrb) [inline requirements.in]: Given time constrains, I am going to open an issue for this. I think we will be able to just drop those requirements in another PR.

Comment (msrb) [inline requirements.in]: https://github.com/fabric8-analytics/fabric8-analytics-server/issues/77

File: Dockerfile (hunk 2 above)
Comment (harjinder-hari) [inline Dockerfile]: We encountered a very weird Import Error with Google Cloud lib. Even if the lib is installed properly and we can see manually that it is available in import-path, but still the program throws Import Error for "protobuf" !

I discussed it in matter-most and Saket/Slavek shared the above work-around.
Reference: https://chat.openshift.io/rh-devtools/pl/5e6sbcsyz3fuxfw1zs4n97hwww
